packages/shex-webapp/doc/shex-simple.js

```diff
@@ -1714,6 +1717,9 @@ function loadSearchParameters () {
         acc.concat(parm + "=" + encodeURIComponent(val)) :
         acc;
     }, []));
+    if ($("#inputData textarea").css("visibility") === "hidden") {
```

Comment:
You could also use `"hideData" in parseQueryString(location.search)` for the conditional. That would avoid depending on the specific css implementation and is potentially a bit more obvious and robust :)

Reply:
I feel like that defeats the purpose of this function – it’s supposed to construct a permanent link based on the current state of the page, not just repeat whatever is in the current URL. At the moment, there page offers no way to toggle the visibility of the input data on the fly, but if that changes, using the query string instead of the actual page state here would result in a confusing bug, I think.